Minor fix in the README (#2)
MeteoSwiss/opendata-forecast-data: README.md

| **Attributes**| **ICON-CH1-EPS** | **ICON-CH2-EPS**|
|-----------|------------------|-----------------|

> | Collection |[ogd-forecasting-icon-ch1](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch1?.language=en) | [ogd-forecasting-icon-ch2](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch2?.language=en) |

Reviewer: ```suggestion
| Collection |[ch.meteoschweiz.ogd-forecasting-icon-ch1](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch1?.language=en) | [ch.meteoschweiz.ogd-forecasting-icon-ch2](https://sys-data.int.bgdi.ch/browser/#/collections/ch.meteoschweiz.ogd-forecasting-icon-ch2?.language=en) |
```

### 2.2 Available Parameters

> Users can find information about available parameters, including metadata, in the collections listed above.

Reviewer: ```suggestion
Users can find information about available parameters, including metadata, in the collection level assets of the above collections.
```

### 2.3 Accessing Forecast Data

> The user can access the forecast model output data from the last 24 hours. Data older than 24 hours is no longer available. To check available data, the user can refer to the relevant collections listed in the table above.

Reviewer: ```suggestion
The user can access the forecast model data from the last 24 hours. Data older than 24 hours is no longer available. The data in each collection is described in the table above.
```
